vuepress/docs/v7.0/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,138 +4,146 @@ sidebarDepth: 2
-
-5. The bundle can now be deployed into the Entando Component Repository with one command
-``` sh
-ent prj deploy
-```
- The `prj deploy` command uses the Git repository URL and project name (e.g. `example-bundle`) to create the custom resource.
-
-Jump to [Install the Bundle into an Application](#install-the-bundle-into-an-application) to finish installing your bundle.
+1. From the project directory, initialize an Entando project with default settings:
+   ``` sh
+   ent prj init
+   ```
+
+2. Initialize the publication system. This requires the URL of an empty Git repository (ending in .git) and your Git credentials.
+   ``` sh
+   ent prj pbs-init
+   ```
+
+3. Publish the bundle to Git. By convention, the first version is assigned the tag `v0.0.1`, but the prefix "v" is optional.
+   ``` sh
+   ent prj pbs-publish
+   ```
+   To quickly push subsequent iterations of the bundle to Git, execute the `ent prj pbs-publish` command. Each push will prompt you to input the corresponding bundle version. You must be consistent with versioning format and alphanumeric precedence to ensure that iterations are listed in the correct order. 
+
+4. Deploy the bundle into the ECR:
+   ``` sh
+   ent prj deploy
+   ```
+   The `prj deploy` command uses the Git repository URL and project name (e.g. `example-bundle`) to create the custom resource.
 
 ### Manual Steps 
-1. Change to the bundle directory if needed
-``` sh
-cd example-bundle/bundle
-```
-
-2. Run the following commands to initialize Git and commit the files
-``` sh
-git init
-git add .
-git commit -m "Init Git repository"
-```
-
-3. Add your remote repository as origin and push the bundle
-``` sh
-git remote add origin https://your/remote/repository.git
-git push -u origin master
-```
+1. From the bundle directory, run the following commands to initialize Git and commit the files:
+   ``` sh
+   git init
+   git add .
+   git commit -m "Init Git repository"
+   ```
+
+2. Add your remote repository as origin and push the bundle:
+   ``` sh
+   git remote add origin https://your/remote/repository.git
+   git push -u origin master
+   ```
+
+3. Publish a Git tag:
+   ``` sh
+   git tag -a "v0.0.1" -m "My first tag"
+   git push --tags
+   ```
+
+5. Create the Kubernetes custom resource for your bundle: 
+   ``` sh
+   ent bundler from-git --name=example-bundle --namespace=entando --repository=https://YOUR-REMOTE-REPOSITORY.git --dry-run > example-bundle.yaml
+   ```
+   A bundle must be published to Git before you can create a custom resource for it. Provide the URL of the bundle's remote Git repository (ends with ".git"). To include a thumbnail for your bundle, use the option  `--thumbnail-file` or `--thumbnail-url`.
```

Comment:
(ends with ".git") >> repeated above

Reply:
no direct repetition. i think this is left over from us wanting to make it really clear that you have to use that particular url form